src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
it's redundant 'cause you specified that property above 
<img width="392" alt="image" src="https://github.com/mate-academy/layout_stars/assets/70220667/a5dd986a-4043-4b84-a90a-8a22e4fc12a7">
```suggestion

```